code/game/objects/obj_defense.dm

```diff
@@ -22,6 +22,8 @@
 
 ///returns the damage value of the attack after processing the obj's various armor protections
 /obj/proc/run_obj_armor(damage_amount, damage_type, damage_flag = 0, attack_dir, armour_penetration = 0)
+	if(damage_flag == "melee" && damage_amount < damage_deflection)
```

Comment:
Would this mean that a shotgun pointblank would do nothing?

Reply:
Shotgun has damage_flag == "bullet".

Reply:
Yeah lmao read it wrong, thought only melee and damage over damaged